src/main/resources/templates/signup.html

```diff
@@ -1,68 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
On your other HTML pages you have `<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">`. While it works without it, consistency is key. Either they should all have it or they shouldn't (I recommend always being more explicit, but others would say otherwise)